### ISSUE TITLE
Unnecessary param in `mlem create deployment`

### ISSUE THREAD

**Issue by aguschin** [iterative/mlem#227]
If you already created an env, why you should specify this again when creating deploy?
Example:
```
$ mlem create env heroku staging  # note HEROKU
💾 Saving env to staging.mlem

$ mlem create deployment heroku myservice -c app_name=mlem-quick-start -c model=rf -c env=staging  # HEROKU again
💾 Saving deployment to myservice.mlem
```

Seems like unnecessary duplication, but maybe I'm missing something.

**Comment by aguschin** [comment]
May be fixed by #363

**Comment by mike0sv** [comment]
It will be partly fixed. It will allow to skip env entirely, but if you want to set up some env params, you will probably still need to specify env type
